Api/Api/Controllers/PredictionController.cs

```diff
@@ -29,28 +35,17 @@ public IActionResult GetTypes()
     }
     
     [HttpPost]
-    public IActionResult Predict([FromQuery]PredictionTypes type, [FromBody]JsonElement input)
+    public IActionResult Predict([FromQuery]int type, [FromBody]JsonElement input)
```

Comment:
Why did you change PredictionTypes to int? I get it that we removed the switch case, but still it seems better to read an Enum than an int variable